src/DIRAC/AccountingSystem/Service/DataStoreHandler.py

```diff
@@ -32,120 +32,61 @@ def initializeHandler(cls, svcInfoDict):
         """
         Register a new type. (Only for all powerful admins)
-        (Bow before me for I am admin! :)
-        """
-        retVal = gConfig.getSections("/DIRAC/Setups")
-        if not retVal["OK"]:
-            return retVal
-        errorsList = []
-        for setup in retVal["Value"]:
-            retVal = self.__acDB.registerType(  # pylint: disable=no-member
-                setup, typeName, definitionKeyFields, definitionAccountingFields, bucketsLength
-            )
-            if not retVal["OK"]:
-                errorsList.append(retVal["Message"])
-        if errorsList:
-            return S_ERROR("Error while registering type:\n %s" % "\n ".join(errorsList))
-        return S_OK()
+        """
+        return self.__acDB.registerType(typeName, definitionKeyFields, definitionAccountingFields, bucketsLength)
 
     types_setBucketsLength = [str, list]
 
     def export_setBucketsLength(self, typeName, bucketsLength):
         """
         Change the buckets Length. (Only for all powerful admins)
-        (Bow before me for I am admin! :)
-        """
-        retVal = gConfig.getSections("/DIRAC/Setups")
-        if not retVal["OK"]:
-            return retVal
-        errorsList = []
-        for setup in retVal["Value"]:
-            retVal = self.__acDB.changeBucketsLength(setup, typeName, bucketsLength)  # pylint: disable=no-member
-            if not retVal["OK"]:
-                errorsList.append(retVal["Message"])
-        if errorsList:
-            return S_ERROR("Error while changing bucketsLength type:\n %s" % "\n ".join(errorsList))
-        return S_OK()
+        """
+        return self.__acDB.changeBucketsLength(typeName, bucketsLength)
 
     types_regenerateBuckets = [str]
 
     def export_regenerateBuckets(self, typeName):
         """
         Recalculate buckets. (Only for all powerful admins)
-        (Bow before me for I am admin! :)
-        """
-        retVal = gConfig.getSections("/DIRAC/Setups")
-        if not retVal["OK"]:
-            return retVal
-        errorsList = []
-        for setup in retVal["Value"]:
-            retVal = self.__acDB.regenerateBuckets(setup, typeName)  # pylint: disable=no-member
-            if not retVal["OK"]:
-                errorsList.append(retVal["Message"])
-        if errorsList:
-            return S_ERROR("Error while recalculating buckets for type:\n %s" % "\n ".join(errorsList))
-        return S_OK()
+        """
+        return self.__acDB.regenerateBuckets(typeName)
 
     types_getRegisteredTypes = []
 
     def export_getRegisteredTypes(self):
         """
         Get a list of registered types (Only for all powerful admins)
-        (Bow before me for I am admin! :)
         """
-        return self.__acDB.getRegisteredTypes()  # pylint: disable=no-member
-
-    types_deleteType = [str]
-
-    def export_deleteType(self, typeName):
```

Comment:
I guess you decided to delete this method because not used anymore?
`AccountingDB.deleteType()` still exists, is it expected?

Reply:
IIRC (several weeks have passes) I left `AccountingDB.deleteType()` because that *might* still be useful (?), but I judged that exposing such method through the service would be just too dangerous.